### PR TITLE
MODFISTO-36 Release 4.0.0

### PR DESCRIPTION
**Pull request by grabsefx**
## Purpose
https://issues.folio.org/browse/MODFISTO-36

## Approach
* Updating NEWS.md, ramls, interface version

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
-## 3.1.0 - Unreleased
+## 4.1.0 - Unreleased
+
+## 4.0.0 - Released
+[Full Changelog](https://github.com/folio-org/mod-finance-storage/compare/v3.0.0...v4.0.0)
+
+### Stories
+* [MODFISTO-59](https://issues.folio.org/browse/MODFISTO-59)  Define and Implement FundType API
```
Comment (pautau-s) [inline NEWS.md]: This story has name - Transaction Calculations - Encumbrance upon update

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,43 @@
...
+* [MODFISTO-50](https://issues.folio.org/browse/MODFISTO-50)  GroupFiscalYearSummary Schema
+* [MODFISTO-49](https://issues.folio.org/browse/MODFISTO-49)  Ledger/LedgerFY updates: Schema changes
+* [MODFISTO-48](https://issues.folio.org/browse/MODFISTO-48)  Adjustments to allow listing budgets by group/fiscalYear/ledger
+* [MODFISTO-46](https://issues.folio.org/browse/MODFISTO-46)  Update transaction schema
+* [MODFISTO-42](https://issues.folio.org/browse/MODFISTO-42)  Finance Records: Inheriting status from parent
+* [MODFISTO-41](https://issues.folio.org/browse/MODFISTO-41)  Data migration scripts for recent schema changes
+* [MODFISTO-38](https://issues.folio.org/browse/MODFISTO-38)  Create GroupFundFY API
+* [MODFISTO-37](https://issues.folio.org/browse/MODFISTO-37)  Update LedgerFY table
+* [MODFISTO-35](https://issues.folio.org/browse/MODFISTO-35)  Update GroupFundFY table
+* [MODFISTO-34](https://issues.folio.org/browse/MODFISTO-34)  Spike: Avoiding partially paid invoices POC
```
Comment (pautau-s) [inline NEWS.md]: There is no changes in code related to this story

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,43 @@
...
+* [MODFISTO-22](https://issues.folio.org/browse/MODFISTO-22)  Create Group schemas
+* [MODFISTO-21](https://issues.folio.org/browse/MODFISTO-21)  Create/Update Ledger schemas
+* [MODFISTO-20](https://issues.folio.org/browse/MODFISTO-20)  Update Transaction schemas
+* [MODFISTO-19](https://issues.folio.org/browse/MODFISTO-19)  Update fund schema
+* [MODFISTO-18](https://issues.folio.org/browse/MODFISTO-18)  Define and Implement FundType API
+* [MODFISTO-2358](https://issues.folio.org/browse/FOLIO-2358)  Use JVM features (UseContainerSupport, MaxRAMPercentage) to manage container memory
+
+### Bug Fixes
+* [MODFISTO-52](https://issues.folio.org/browse/MODFISTO-52)  Unavailable increasing when allocating from budget to budget
+* [MODFISTO-43](https://issues.folio.org/browse/MODFISTO-43)  Return error if specified Fund does not have a budget in the current fiscal year upon transaction processing
```
Comment (pautau-s) [inline NEWS.md]: This story is closed as duplicate

File: NEWS.md [hunk 4]
```diff
@@ -1,4 +1,43 @@
-## 3.1.0 - Unreleased
+## 4.1.0 - Unreleased
+
+## 4.0.0 - Released
```
Comment (khandramai) [inline NEWS.md]: Maybe it makes sense to add short description about what was done before this release? Smth like implemented APIs: ...

**Pull request merged**